### PR TITLE
add next article

### PR DESCRIPTION
**Pull request by codiebeulaine**
This PR adds a 'next article' to an article so that when you are viewing an article, it could navigate you to the next article within a section, if one exists.

### CODE REVIEW

**Review by KaitCrawford — APPROVED** [review]
:+1:

File: articles/templatetags/article_tags.py [hunk 1]
```diff
@@ -16,3 +16,13 @@ def footer_pages(context):
         "request": request,
         "footers": pages,
     }
+
+
+@register.simple_tag(takes_context=True)
+def get_next_article(context, article):
+    return (
+        article.specific.get_siblings(False)
+        .filter(path__gte=article.path, live=True, content_type=8)
```
Comment (zerolab) [inline articles/templatetags/article_tags.py:25]: suggestion: avoid hardcoding as much as possible. content_type 8 reads like a magiv value.

e.g. make use of [`exact_type`](https://docs.wagtail.io/en/stable/reference/pages/queryset_reference.html#wagtail.core.query.PageQuerySet.exact_type)

The query could look like:

`Page.objects.exact_type(ArticlePage).siblings_of(article).filter(path__gte=self.path).live().specific().order_by('path')`

**Review by zerolab — COMMENTED** [review]
I have one suggestion for `get_next_article`

File: articles/templatetags/article_tags.py (hunk 1 above)
Comment (codiebeulaine) [reply articles/templatetags/article_tags.py]: 👍

**Pull request merged**